pkg/driver_cryptocell_310/Makefile

```diff
@@ -1,36 +1,42 @@
-PKG_NAME=driver_cryptocell_310
-PKG_URL=https://www.nordicsemi.com/-/media/Software-and-other-downloads/SDKs/nRF5/Binaries
-PKG_VERSION=17.1.0
-PKG_EXT=zip
-PKG_DIR_NAME=nRF5_SDK_17.1.0_ddde560
-PKG_LICENSE=ARM Object Code and Header Files License
+PKG_NAME      = driver_cryptocell_310
+PKG_URL       = https://www.nordicsemi.com/-/media/Software-and-other-downloads/SDKs/nRF5/Binaries
+PKG_VERSION   = 17.1.0_ddde560
+PKG_DIR_NAME  = nRF5_SDK
+PKG_EXT       = zip
+PKG_LICENSE   = ARM Object Code and Header Files License
+PKG_SHA512  	= cb610d724d27ef79c792511d8b889cbf2d411a3a391f5c740ea10b86139201b97eedf0c1edcf4d199c3fbd51e4cc5a4bd922c0c40d01e2f5359acd16a2b163e4
 
 PKG_SOURCE_DIR ?= $(PKGDIRBASE)/$(PKG_NAME)
-NRF_CC310_PATH = $(PKG_DIR_NAME)/external/nrf_cc310
-PKG_ZIPFILE = $(PKG_DIR_NAME).$(PKG_EXT)
+PKG_ZIP_DIR = $(PKGDIRBASE)/$(PKG_DIR_NAME)_$(PKG_VERSION)
+PKG_ZIPFILE = $(PKGDIRBASE)/$(PKG_DIR_NAME)_$(PKG_VERSION).$(PKG_EXT)
+PKG_ZIP_URL = $(PKG_URL)/$(PKG_DIR_NAME)_$(PKG_VERSION).$(PKG_EXT)
+
+NRF_CC310_PATH = external/nrf_cc310
 
 ifneq ($(RIOTBASE),)
 include $(RIOTBASE)/Makefile.base
 endif
 
-.PHONY: all clean distclean prepare
+.PHONY: all clean distcleanq
```

Comment:
Is `distcleanq` supposed to be `distclean`?

Reply:
Yes, obviously, good catch. Unfortunately this got already merged with #19959, but I've added another commit fixing the typo here.